backend/services.endorsement-reminder/appsettings.json

```diff
@@ -14,5 +14,7 @@
     "Url": "localhost",
     "Port": 1025,
     "EnableSsl": false
-  }
+  },
+  "LicensedInitiatedLink": "https://www.doctorsofbc.ca/sites/default/files/endorsement_licensed_initiated_4.pdf",
+  "UnlicensedInitiatedLink": "https://www.doctorsofbc.ca/sites/default/files/endorsement_unlicensed_initiated_4.pdf"
```

Comment:
`appsettings.json` and `PidpConfiguration` are used for values that change between environments (local, dev, test, prod), or for secrets that we don't want to commit into our source code. This URL won't be changing between different environments so I don't think it needs to be here; it can just be directly in the email or in a constant.